### ISSUE TITLE
Nether very low FPS

### ISSUE THREAD

**Issue by ShaneCourtrille** [LunaPixelStudios/Better-Minecraft#26]
### Describe the Bug

When in an area of the nether that has a large number of the BetterNether leaves coming down I'm experiencing very low FPS, to the point that it's causing motion sickness and making the area unplayable (even with a lower distance setting).

There is a way to lower foliage called global_plant_and_structures_density but that appears to also impact structures which don't appear to be the main cause of the problem so I'm hesitant to modify this value on our server.

Looking at their GitHub issues the performance solution seems to be to use Sodium but since the use of shaders features prominently in this pack I don't think that's going to work.

### Reproduction Steps

1.
2.
3.
...

### Operating System

Windows 10

### Minecraft Version

1.18

### Modloader

Fabric

### Modpack Version

9

### Optifine

No

**Comment by switchnlade** [comment]
Hey there, just for some info, our fabric packs include Sodium already, the usage of Iris doesn't actually interfere with that. How much ram do you have allocated to the game and how much is total in your pc? Sometimes that's the issue so I just want to double check it.

**Comment by ShaneCourtrille** [comment]
32 GB available but I haven't increased allocation.  Let me try that and
get back to you.  Thanks!

**Comment by SharkieTV** [comment]
> 

Allocate at least 6gb-8gb of ram and see if removing dynamic lights reforged helps.

**Comment by Lucipo** [comment]
Also having this issue with a 10 chunk render distance and the increased vertical height of the betternether to 256 blocks tall of generation. The lag is only seen in the upsidedown forest biome, with the insane amount of glowing leaves tanking my game to 0 fps as more chunks are loaded.  4 chunk render distance is same issue. With Sildurs vibrant off, there is still lag but not as detrimental as 0 fps gameplay.

**Comment by SharkieTV** [comment]
Im not getting this issue unfortunately, have you tried removing the lambda dynamic lights mod?

**Comment by ShaneCourtrille** [comment]
Yeah, that's the name of the biome and it causes a problem for everyone
playing on our server.  I've got 8GB allocated now and still have the same
issue even after getting rid of dynamic lights.  Going to just remove the
mod from our server as there doesn't appear to be a way to remove that
biome.

**Comment by SharkieTV** [comment]
Sodium is already in the pack and the biome nothing to do with the server, only with the players pc performance. In the BCLib folder you can disable the Nether biome if you wish, other than that there isn't much else I can help with.

**Comment by ShaneCourtrille** [comment]
Unfortunately you cannot currently disable biomes as per
https://github.com/paulevsGitch/BCLib/issues/63.  I'll just pull the mod
since based on that BCLib discussion there look to be quite a few
problematic biomes in that mod pack.

Thanks.
